install.md

```diff
@@ -1,5 +1,36 @@
 # libpod Installation Instructions
 
+## Installing packaged versions of Podman
+
+#### Arch Linux
+
+https://aur.archlinux.org/packages/libpod/
+
+#### Fedora, Centos, OpenSUSE, SUSE:
+
+sudo yum -y install podman
```

Comment:
Podman is not (yet) supported on SUSE SLES, so we should not list it.

I recommend creating a separate section for `openSUSE` (lower 'o') and install it via `sudo zypper install podman` (no yum).